pandas/core/indexes/frozen.py

```diff
@@ -9,10 +9,15 @@
 """
 
 import warnings
+
 import numpy as np
-from pandas.core.base import PandasObject
+
 from pandas.util._decorators import deprecate_kwarg
+
```

Comment:
Hmm what's with the empty lines between `from` statements here? Not seeing that anywhere else?

Reply:
These are separate sections see the isort section in the setup.cfg for latest spec.